app/api/chemotion/attachment_api.rb

```diff
@@ -44,6 +44,90 @@ def upload_chunk_error_message
       error!(message, 404)
     end
 
+    resource :inbox do
```

Comment:
Please remove this inbox endpoint from here. It has been moved to the proper location (aka in the InboxAPI)
See: https://github.com/ComPlat/chemotion_ELN/blob/development-5/app/api/chemotion/inbox_api.rb#L9